### ISSUE TITLE
Consistent match all character class

### ISSUE THREAD

**Issue by RunDevelopment** [RunDevelopment/eslint-plugin-clean-regex#9]
There are multiple ways to express a character class which accepts all characters but you should choose one and stick with it to make your regexes easier to understand.

Examples:

```js
/[\s\S]/, /[\d\D]/, /[\w\W]/, /[^]/
```

_Notes:_
If present, we could even take advantage of the `s` flag.

This should also detect alternations which are equal to the set of all characters. Examples:

```js
/(?:\s|\S)/, /\s|\S/, /(?:.|\s)/, /.|\D/
```
